Create Help article “verify-your-identity/verify-your-identity-in-person” (#948)
18F/identity-site: content/_help/verify-your-identity/verify-your-identity-in-person._fr.md

---
title: Vérifier votre identité en personne
category: verify-your-identity
permalink: /fr/help/verify-your-identity/verify-your-identity-in-person/
order: 7
---

> Si vous ne parvenez pas à télécharger avec succès votre identifiant sur Login.gov, vous pourrez peut-être le vérifier en personne auprès de nos partenaires du Service postal des États-Unis (USPS). Pour le moment, nous n'offrons cette option que dans certains [bureaux USPS de la région de Washington (District de Columbia)](https://docs.google.com/document/d/1vX2VAskfRxzhEttJDiCJwi88czZpfSYa/edit#heading=h.1t3h5sf) et pour certaines agences partenaires.

Reviewer: In-page heading for French.

```suggestion
Si vous ne parvenez pas à télécharger avec succès votre identifiant sur Login.gov, vous pourrez peut-être le vérifier en personne auprès de nos partenaires du Service postal des États-Unis (USPS). Pour le moment, nous n'offrons cette option que dans certains [bureaux USPS de la région de Washington (District de Columbia)](#sites-disponibles-dans-la-région-de-washington-d-c) et pour certaines agences partenaires.
```

## Commencez cette procédure sur Login.gov avant de vous rendre au bureau de poste 

Veuillez préparer les informations suivantes pour les saisir sur Login.gov :

> 1. Votre carte d'identité délivrée par l'État. [Vous n'avez pas de carte d'identité délivrée par l'État?](https://login.gov/help/verify-your-identity/accepted-state-issued-identification/)

Reviewer: Root-relative URLs. Should also maintain the language path prefix.

```suggestion
1. Votre carte d'identité délivrée par l'État. [Vous n'avez pas de carte d'identité délivrée par l'État?](/fr/help/verify-your-identity/accepted-state-issued-identification/)
```

En attendant, veuillez contacter le centre d'assistance de l'agence partenaire pour savoir ce que vous pouvez faire.

### Preuve de votre adresse actuelle

Si votre adresse actuelle ne correspond pas à l'adresse figurant sur votre pièce d'identité, vous devrez apporter une preuve de votre adresse actuelle au bureau de poste. Les formes acceptables de preuve d'adresse sont :

* Bail, hypothèque ou acte de fiducie
* Inscription sur les listes électorales
* Carte d'immatriculation d’un véhicule
* Police d'assurance habitation ou véhicule

Vous pouvez apporter une copie physique ou montrer une copie numérique du document. Vous ne pouvez pas montrer une photo ou une capture d'écran du document. 

## Effectuez le processus au bureau de poste avant la date limite

### Sites disponibles dans la région de Washington D. C. 

Vous pouvez vous rendre dans l'un des bureaux de poste suivants. Vous n'avez pas besoin de vous rendre à l'emplacement que vous avez sélectionné sur Login.gov. 

> 1. BALTIMORE — Post Office™\
>    900 E FAYETTE ST RM 118\
>    BALTIMORE, MD 21233-9715
> 2. BETHESDA — Post Office™\
>    6900 WISCONSIN AVE STE 100\
>    CHEVY CHASE, MD 20815-9996
> 3. FRIENDSHIP — Post Office™\
>    4005 WISCONSIN AVE NW\
>    WASHINGTON, D. C. 20016-9997
> 4. WASHINGTON — Post Office™\
>    900 BRENTWOOD RD NE\
>    WASHINGTON, D. C. 20018-9997
> 5. ARLINGTON — Post Office™\
>    3118 WASHINGTON BLVD\
>    ARLINGTON, VA 22201-9998
> 6. ASHBURN — Post Office™\
>    44715 PRENTICE DR\
>    DULLES, VA 20101-9996
> 7. MERRIFIELD — Post Office™\
>    8409 LEE HWY\
>    MERRIFIELD, VA 22116-9998

Reviewer: Same question about bolded names, for French. Also, these slashes "\\" will appear literally in the page content (it's [a Netlify CMS issue](https://github.com/netlify/netlify-cms/issues/512)).

```suggestion
1. **BALTIMORE — Post Office™**
   900 E FAYETTE ST RM 118
   BALTIMORE, MD 21233-9715
2. **BETHESDA — Post Office™**
   6900 WISCONSIN AVE STE 100
   CHEVY CHASE, MD 20815-9996
3. **FRIENDSHIP — Post Office™**
   4005 WISCONSIN AVE NW
   WASHINGTON, D. C. 20016-9997
4. **WASHINGTON — Post Office™**
   900 BRENTWOOD RD NE
   WASHINGTON, D. C. 20018-9997
5. **ARLINGTON — Post Office™**
   3118 WASHINGTON BLVD
   ARLINGTON, VA 22201-9998
6. **ASHBURN — Post Office™**
   44715 PRENTICE DR
   DULLES, VA 20101-9996
7. **MERRIFIELD — Post Office™**
   8409 LEE HWY
   MERRIFIELD, VA 22116-9998
```

Author: AH, thought I'd removed the slashes, thank you!

* Essayer à nouveau de vérifier votre identité en ligne.
* Contacter le centre d'assistance de l'agence partenaire pour obtenir de l'aide.

Si vous ne recevez pas de courriel de notre part, il y a quelques étapes que vous pouvez suivre :

* Vérifiez dans le dossier de pourriels ou de courriers indésirables de votre courriel s'il y a un courriel de [no-reply@login.gov](mailto:no-reply@login.gov).
* Si vous n'avez pas eu de nouvelles après 24 heures de votre visite au bureau de poste, veuillez contacter le service clientèle de Login.gov sur le site login.gov/contact.

# Résolution des problèmes liés à la vérification de votre identité en personne

Si vous avez rencontré un problème en essayant de vérifier votre identité en personne, nous aimerions vous aider à le résoudre. Voici quelques conseils pour résoudre le problème. 

## Ce qu’il faut faire si vous avez reçu un courriel, mais que vous n’avez pas essayé de vérifier votre identité en personne

Si vous avez reçu un courriel vous informant que vous êtes prêt à vérifier votre identité ou que votre identité n’a pas pu être vérifiée en personne, et que vous n’avez pas tenté cette action: 

> [Connectez-vous](https://secure.login.gov/) et [changez votre mot de passe immédiatement](https://login.gov/help/manage-your-account/change-your-password/). Si vous le souhaitez, vous pouvez également [supprimer votre compte](https://login.gov/help/manage-your-account/delete-your-account/).

Reviewer: Root-relative URLs. Should also maintain the language path prefix.

```suggestion
[Connectez-vous](https://secure.login.gov/fr/) et [changez votre mot de passe immédiatement](/fr/help/manage-your-account/change-your-password/). Si vous le souhaitez, vous pouvez également [supprimer votre compte](/fr/help/manage-your-account/delete-your-account/).
```

## Ce qu’il faut faire si vous n’avez pas pu vérifier votre identité en personne

Si vous ne parvenez pas à vérifier votre identité en personne, vous pouvez réessayer de [vérifier votre identité en ligne](/fr/help/verify-your-identity/how-to-verify-your-identity/) ou contacter l’agence à laquelle vous essayez d’accéder [pour obtenir de l’aide sur des questions spécifiques](/fr/help/specific-agencies/overview/) relatives la vérification de votre identité.

> ### Conseils pour vérifier votre identité en personne: 

Reviewer: This isn't a heading in the other languages. Do we mean for it to be in the French version?

```suggestion
Conseils pour vérifier votre identité en personne: 
```

Author: Haha I just saw this too! Should be fixed in the latest commit.

> Si vous ne parvenez pas à vérifier votre identité en personne, vous pouvez réessayer de [vérifier votre identité en ligne](https://login.gov/help/verify-your-identity/how-to-verify-your-identity/) ou contacter l’agence à laquelle vous essayez d’accéder [pour obtenir de l’aide sur des questions spécifiques](https://login.gov/help/specific-agencies/overview/) relatives la vérification de votre identité.

Reviewer: Root-relative URLs. Should also maintain the language path prefix.

```suggestion
Si vous ne parvenez pas à vérifier votre identité en personne, vous pouvez réessayer de [vérifier votre identité en ligne](/fr/help/verify-your-identity/how-to-verify-your-identity/) ou contacter l’agence à laquelle vous essayez d’accéder [pour obtenir de l’aide sur des questions spécifiques](/fr/help/specific-agencies/overview/) relatives la vérification de votre identité.
```

Votre pièce d’identité ou votre permis de conduire délivré par l’État doit être en cours de validité. Nous n’acceptons actuellement aucune autre forme d’identification.

Apportez un justificatif de domicile valide si votre adresse actuelle est différente de celle qui figure sur votre pièce d’identité. Pour vérifier votre adresse actuelle, vous pouvez apporter votre:

* Bail, hypothèque ou acte de fiducie
* Inscription sur les listes électorales
* Carte d’immatriculation de véhicule
* Police d’assurance habitation ou automobile

## Ce qu’il faut faire si le délai pour vérifier votre identité en personne est passé

> Si vous avez reçu un courriel vous informant que le délai de vérification de votre identité est expiré, essayez [à nouveau de vérifier votre identité en ligne](https://login.gov/help/verify-your-identity/how-to-verify-your-identity/). 

Reviewer: Root-relative URLs. Should also maintain the language path prefix.

```suggestion
Si vous avez reçu un courriel vous informant que le délai de vérification de votre identité est expiré, essayez [à nouveau de vérifier votre identité en ligne](/fr/help/verify-your-identity/how-to-verify-your-identity/). 
```

Nous annulons automatiquement les demandes après 30 jours afin de protéger les données personnelles des demandeurs. Lorsque vous relancerez le processus de vérification de votre identité en personne, vous recevrez une nouvelle date limite.

> ### Artículos relacionados

Reviewer: Currently this "Related Articles" would be a subheading of "What to do if the deadline to verify your identity in person passed" .

```suggestion
## Artículos relacionados
```

> [Aide pour des agences spécifiques](https://login.gov/help/specific-agencies/overview/)
> 
> [Pièces d’identité acceptées, délivrées par l’État](https://login.gov/help/verify-your-identity/accepted-state-issued-identification/)
> 
> [Problèmes lors du téléversement de votre pièce d’identité délivrée par l’État](https://login.gov/help/verify-your-identity/troubleshoot-uploading-your-state-issued-id/)
> 
> [Vérifier votre identité en ligne](https://login.gov/help/verify-your-identity/how-to-verify-your-identity/)

Reviewer: ```suggestion
[Aide pour des agences spécifiques](/fr/help/specific-agencies/overview/)

[Pièces d’identité acceptées, délivrées par l’État](/fr/help/verify-your-identity/accepted-state-issued-identification/)

[Problèmes lors du téléversement de votre pièce d’identité délivrée par l’État](/fr/help/verify-your-identity/troubleshoot-uploading-your-state-issued-id/)

[Vérifier votre identité en ligne](/fr/help/verify-your-identity/how-to-verify-your-identity/)
```
